lib/modules/datasource/nuget/index.spec.ts

```diff
@@ -14,6 +18,9 @@ const hostRules: any = _hostRules;
 
 jest.mock('../../../util/host-rules', () => mockDeep());
 
+jest.mock('../../../util/cache/package');
```

Comment:
```suggestion
jest.mock('../../../util/cache/package', () => mockDeep());
```
test perf, will block jest to load all drips recursive to create a mock.